resolwe_bio/descriptors/super_sample.yml

```diff
@@ -0,0 +1,263 @@
+- slug: super_sample
+  name: Super sample annotation
+  description: Sample meta data compliant with GEO and ENCODE databases.
+  version: 0.0.1
+  schema:
+    - name: assay_type
+      label: Assay type
+      type: basic:string
+      choices:
+        - label: RNA-seq
+          value: rna-seq
+        - label: ChIP-seq
+          value: chip-seq
+        - label: ATAC-seq
+          value: atac-seq
+        - label: ChIPmentation
+          value: chipmentation
+    - name: biosample_type
```

Comment:
This choices list does not allow for proper classification of other possible types of samples, most notably microorganism cultures, but also other possible types that we might encounter down the road (e.g. metagenomic samples).